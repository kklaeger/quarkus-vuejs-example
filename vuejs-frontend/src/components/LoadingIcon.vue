<template>
  <b-loading :is-full-page="false" v-model="isLoading"></b-loading>
</template>

<script lang="ts">
import { Component, Vue } from "vue-property-decorator";
@Component({
  components: {}
})
export default class LoadingIcon extends Vue {
  private isLoading = true;
}
</script>

<style scoped lang="scss"></style>
