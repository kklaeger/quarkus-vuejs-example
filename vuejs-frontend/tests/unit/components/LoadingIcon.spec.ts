import { shallowMount, createLocalVue } from "@vue/test-utils";
import Buefy from "buefy";
import LoadingIcon from "@/components/LoadingIcon.vue";

const localVue = createLocalVue();
localVue.use(Buefy);

describe(LoadingIcon.name + " component", () => {
  it("should render successfully.", () => {
    const wrapper = shallowMount(LoadingIcon, { localVue });
    expect(wrapper).toMatchSnapshot();
  });
});
