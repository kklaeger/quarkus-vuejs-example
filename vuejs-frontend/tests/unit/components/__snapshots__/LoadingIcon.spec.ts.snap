// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIcon component should render successfully. 1`] = `<b-loading-stub active="true" animation="fade" oncancel="function _default() {}"></b-loading-stub>`;
